website/addons/googledrive/utils.py

```diff
@@ -27,6 +28,7 @@ def to_hgrid(item, node, path):
         'kind': 'folder',
         'name': safe_name,
         'addon': 'googledrive',
+        'provider': 'googledrive',  # TODO: remove duplication -- APIv2
```

Comment:
Particularly given that we're adding APIv2 endpoints, what is the advantage of retrofitting an older method with a new field?